io.github.slgobinath.SafeEyes.yaml

```diff
@@ -33,23 +33,18 @@ cleanup:
   - /lib/*.la
 
 modules:
-  - shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json # app indicator support
-  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
-
-  # alsa-utils, dbus-python, libnotify, xprop -> anitya configured.
+  # alsa-utils, xprop -> anitya configured.
   # wlrctl, xprintidle -> manual checks
-  - modules/python3-requirements.yml # auto generated pip dependencies for safeeyes
-  - modules/dbus-python.yml # dbus - for tray
   - modules/xprop.yml # required for 'xorg do not disturb' plugin
   - modules/wlrctl.yml # required for 'wayland do not disturb' plugin (x86_64 only)
   - modules/alsa-utils.yml # required for 'audible alert' plugin
-  - modules/libnotify.yml # notification support
   - modules/xprintidle.yml # idle detection other than gnome
+  - modules/python3-requirements.yml # auto generated pip dependencies for safeeyes
```

Comment:
This was moved to the last entry. The order here does not seem to make a difference, except that when building, it will rebuild all following entry after a changed one.
When developing with local changes in upstream safeeyes, this means that xprop, etc. would previously get rebuilt unnecessary, where now only everything in python3-requirements.yml needs to be rebuilt.